contracts/Rates.sol

```diff
@@ -35,7 +32,7 @@ contract Rates is Restricted {
     }
 
     function setMultipleRates(bytes32[] symbols, uint[] newRates) external restrict("RatesFeeder") {
-        require(symbols.length == newRates.length, "symobls and newRates lengths must be equal");
+        require(symbols.length == newRates.length, "symbols and newRates lengths must be equal");
```

Comment:
Do we really want to redeploy Rates contract just for this change? No strong opinion about it just double checking.

Reply:
No :)

Reply:
certainly not